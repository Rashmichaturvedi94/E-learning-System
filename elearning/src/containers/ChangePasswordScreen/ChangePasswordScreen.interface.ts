export interface ChangePasswordScreenProps {
  // TODO: add required props below
  myProp?: boolean;
}
