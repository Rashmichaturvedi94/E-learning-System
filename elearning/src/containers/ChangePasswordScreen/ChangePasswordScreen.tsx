import React, { FC } from 'react';
import { Container, Input, TitleText, ContinueButton, SaveText, InfoText  } from './ChangePasswordScreen.styles';
import { ChangePasswordScreenProps } from './ChangePasswordScreen.interface';
import { useNavigation } from '@react-navigation/native';
import { View, Alert } from 'react-native';

export const ChangePasswordScreen: FC<ChangePasswordScreenProps>  = () => {
  const navigation = useNavigation();
  return (
    <Container>
      <View>
        <TitleText>Change Password</TitleText>
        <Input placeholder="Old Password" />
        <Input placeholder="New Password" />
        <Input placeholder="Confirm Password" />
        <ContinueButton onPress={() => Alert.alert('Simple Button pressed')}>
          <SaveText>Continue</SaveText>
        </ContinueButton>
       
      </View>
    </Container>
  );
};
