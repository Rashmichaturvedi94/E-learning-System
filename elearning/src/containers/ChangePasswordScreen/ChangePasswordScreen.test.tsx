import React from 'react';
import { render } from '@testing-library/react-native';
import { ChangePasswordScreen } from './ChangePasswordScreen';

describe('containers/ChangePasswordScreen', () => {
  it('should render', () => {
    const component = render(<ChangePasswordScreen />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
