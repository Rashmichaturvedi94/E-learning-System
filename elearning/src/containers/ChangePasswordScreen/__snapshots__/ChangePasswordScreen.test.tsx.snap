// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers/ChangePasswordScreen should render 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Array [
      Object {
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": 0,
        "marginLeft": 16,
        "marginRight": 16,
        "marginTop": 0,
      },
    ]
  }
>
  <View>
    <Text
      style={
        Array [
          Object {
            "fontSize": 30,
            "fontWeight": "bold",
            "marginBottom": 50,
            "textAlign": "center",
          },
        ]
      }
    >
      Change Password
    </Text>
    <TextInput
      allowFontScaling={true}
      placeholder="Old Password"
      rejectResponderTermination={true}
      style={
        Array [
          Object {
            "borderBottomColor": "#b0adac",
            "borderBottomWidth": 0.5,
            "borderColor": "white",
            "height": 40,
            "marginTop": 20,
          },
        ]
      }
      underlineColorAndroid="transparent"
    />
    <TextInput
      allowFontScaling={true}
      placeholder="New Password"
      rejectResponderTermination={true}
      style={
        Array [
          Object {
            "borderBottomColor": "#b0adac",
            "borderBottomWidth": 0.5,
            "borderColor": "white",
            "height": 40,
            "marginTop": 20,
          },
        ]
      }
      underlineColorAndroid="transparent"
    />
    <TextInput
      allowFontScaling={true}
      placeholder="Confirm Password"
      rejectResponderTermination={true}
      style={
        Array [
          Object {
            "borderBottomColor": "#b0adac",
            "borderBottomWidth": 0.5,
            "borderColor": "white",
            "height": 40,
            "marginTop": 20,
          },
        ]
      }
      underlineColorAndroid="transparent"
    />
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "#1b8ffa",
            "borderRadius": 10,
            "height": 50,
            "justifyContent": "center",
            "marginTop": 100,
          },
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#ffffff",
            },
          ]
        }
      >
        Continue
      </Text>
    </View>
  </View>
</RCTSafeAreaView>
`;
